features/steps/api_component.go

```diff
@@ -99,11 +99,9 @@ func NewInteractivesApiComponent(mongoURI string) (*InteractivesApiComponent, er
 	log.Info(context.Background(), "configuration for component test", log.Data{"config": c.Config})
 
 	cfg, _ := config.Get()
-	mongodb := &mongo.Mongo{
-		URI:        strings.Replace(mongoURI, "mongodb://", "", 1),
-		Database:   cfg.MongoConfig.Database,
-		Collection: cfg.MongoConfig.Collection,
-	}
+	cfg.MongoConfig.BindAddr = strings.Replace(mongoURI, "mongodb://", "", 1)
+	cfg.MongoConfig.Database = "interactives-api"
+	mongodb := &mongo.Mongo{Config: cfg}
```

Comment:
Is this using a real mongo db database for the tests or a mock/stubbed one

Reply:
real 👍 inmem from here: https://github.com/ONSdigital/dp-component-test/blob/f11d339771c967481617d27cb359ba6f9312ccaa/mongo_feature.go#L10 - ref in api: https://github.com/ONSdigital/dp-interactives-api/blob/bc814e66e603005725fc5c2e6e49b2e6f9685e70/main_test.go#L64

Reply:
ideally we should use real everything from docker - i think static files are using similar strategy - im gonna add a ticket for this 😊

Reply:
https://trello.com/c/DyFdTaXn/171-techdebt-better-component-tests-not-using-mocks

Reply:
Mocked/stubbed 3rd party services is the standard approach in the ONS projects I believe.
I think they are looking at using real services when they start trying to implement integration testing.